lib/src/reflectable_builder_based.dart

```diff
@@ -172,14 +170,12 @@ class ReflectorData {
 Map<Reflectable, ReflectorData> data =
     throw new StateError("Reflectable has not been initialized. "
         "Did you forget to add the main file to the "
-        "reflectable transformer's entry_points in pubspec.yaml?");
+        "reflectable builder's entry_points in pubspec.yaml?");
 
 /// This mapping translates symbols to strings for the covered members.
 /// It will be initialized in the generated code.
 Map<Symbol, String> memberSymbolMap =
-    throw new StateError("Reflectable has not been initialized. "
-        "Did you forget to add the main file to the "
-        "reflectable transformer's entry_points in pubspec.yaml?");
+    throw new StateError("Reflectable has not been initialized.");
```

Comment:
This message seems less informative now. Could we add some kind of advice for how to initialize reflectable?

Reply:
Adjusted both messages to be identical (using a single `const String`), giving the advice that `main` should call `initializeReflectable()` as its first action, which would presumably be the typical (and only?) way any developer could end up encountering the situation where those `StateError`s are thrown.

If no code is generated we will have a compile-time error (because there is an import for something that doesn't exist), so the old reason for having this does not exist any more.

If there is no import of the generated file (whether or not it's present) and no `initializeReflectable()` then we may still have other libraries in the same program that import reflectable-something, and in that case they will actually still hit the above `StateError`, so I think we're OK in all cases! ;-)